test_backup_restore_new/test_cancel_backup.py::test_cancel_backup is flaky

https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=77402&sha=10dc89e6405bce5fe7481e94ba86828fb2f8aeef&name_0=PR&name_1=Integration%20tests%20%28aarch64%2C%203%2F4%29

```
=================================== FAILURES ===================================
______________________________ test_cancel_backup ______________________________
[gw1] linux -- Python 3.10.12 /usr/bin/python3

    def test_cancel_backup():
        # We use partitioning so backups would contain more files.
        node.query(
            "CREATE TABLE tbl (x UInt64) ENGINE=MergeTree() ORDER BY tuple() PARTITION BY x%20"
        )
    
        node.query(f"INSERT INTO tbl SELECT number FROM numbers(500)")
    
        try_backup_id_1 = uuid.uuid4().hex
        start_backup(try_backup_id_1)
>       cancel_backup(try_backup_id_1)

test_backup_restore_new/test_cancel_backup.py:188: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

backup_id = 'd06884d8da2545f99c7f8f411699243e'

    def cancel_backup(backup_id):
        node.query(
            f"KILL QUERY WHERE query_kind='Backup' AND query LIKE '%{backup_id}%' SYNC"
        )
        assert (
            node.query(f"SELECT status FROM system.backups WHERE id='{backup_id}'")
            == "BACKUP_CANCELLED\n"
        )
        expected_error = "QUERY_WAS_CANCELLED"
        assert expected_error in node.query(
            f"SELECT error FROM system.backups WHERE id='{backup_id}'"
        )
        assert (
            node.query(
                f"SELECT count() FROM system.processes WHERE query_kind='Backup' AND query LIKE '%{backup_id}%'"
            )
            == "0\n"
        )
        node.query("SYSTEM FLUSH LOGS")
        kill_duration_ms = int(
            node.query(
                f"SELECT query_duration_ms FROM system.query_log WHERE query_kind='KillQuery' AND query LIKE '%{backup_id}%' AND type='QueryFinish'"
            )
        )
>       assert kill_duration_ms < 2000  # Query must be cancelled quickly
E       assert 2095 < 2000

test_backup_restore_new/test_cancel_backup.py:108: AssertionError
---------------------------- Captured stdout setup -----------------------------
```


Maybe 2 seconds is not enough

--- a/tests/integration/test_backup_restore_new/test_cancel_backup.py
+++ b/tests/integration/test_backup_restore_new/test_cancel_backup.py
@@ -105,7 +105,13 @@ def cancel_backup(backup_id):
             f"SELECT query_duration_ms FROM system.query_log WHERE query_kind='KillQuery' AND query LIKE '%{backup_id}%' AND type='QueryFinish'"
         )
     )
-    assert kill_duration_ms < 2000  # Query must be cancelled quickly
+    # Common failures in CI with 2000:
+    # E       assert 2520 < 2000
+    # E       assert 2210 < 2000
+    # E       assert 2160 < 2000
+    # E       assert 2143 < 2000
+    # E       assert 2495 < 2000
+    assert kill_duration_ms < 4000  # Query must be cancelled quickly
 
 
 # Start restoring from a backup.

